std_msgs/String node_name
std_msgs/Bool node_status
std_msgs/Int32 node_id